test/setup.ts

```diff
@@ -1,19 +1,19 @@
 import { expect } from "chai";
 import { deployments } from "hardhat";
 
-import { HypercertMinterV0 } from "../src/types";
-import { ImpactScopes, Rights, WorkScopes } from "./wellKnown";
+import { HypercertMinterV0, HypercertMinterV1 } from "../src/types";
+import { HypercertMinter_V0, HypercertMinter_V1, ImpactScopes, Rights, WorkScopes } from "./wellKnown";
 
-export type AddressedHypercertMinterV0 = {
+export type AddressedHypercertMinterV1 = {
   address: string;
-  minter: HypercertMinterV0;
+  minter: HypercertMinterV1;
 };
 
 export type HypercertCollection = {
-  minter: HypercertMinterV0;
-  deployer: AddressedHypercertMinterV0;
-  user: AddressedHypercertMinterV0;
-  anon: AddressedHypercertMinterV0;
+  minter: HypercertMinterV1;
```

Comment:
Perhaps here also as below worth defining a "pointer" e.g. HypercertMinterCurrentVersion or somesuch and then you just have one place to update..?